### PR TITLE
Have an instance of Permissions on Navigator and WorkerNavigator.

### PR DESCRIPTION
**Pull request by mounirlamouri**
Closes issue 11.

@marcoscaceres PTAL?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -412,12 +412,31 @@
     </section>
     <section>
       <h2>
+        Navigator and WorkerNavigator extension
+      </h2>
+      <p>
+        A <a>Permissions</a> instance is exposed on the <code>navigator</code>
+        object for <code>Window</code> and <code>Worker</code> contexts.
+      </p>
+      <dl class='idl' title='[Exposed=(Window)] partial interface Navigator'>
```
Comment (marcoscaceres) [inline index.html]: sorry, why do you have two definitions for Exposed? Don't you just want one, with `[Expose=(Window,Worker)]`?

File: index.html [hunk 2]
```diff
@@ -412,12 +412,31 @@
...
+        <dt>
+          readonly attribute Permissions permissions;
+        </dt>
+      </dl>
+      <dl class='idl' title=
```
Comment (marcoscaceres) [inline index.html]: as above, do you need this redefinition just for the Worker scope?

Comment (marcoscaceres) [inline index.html]: oh, it's WorkerNavigator! d'uh.

File: index.html [hunk 3]
```diff
@@ -445,8 +459,8 @@
         If a developer wants to check multiple permissions at once, the editors
         recommend them to use Promises.all(). It should yield to the same
         result and allow this API to stay simple. If it happens to be a very
-        common use case, it should be easy to extend Permissions.query() to
-        accept a sequence&lt;&gt; too.
+        common use case, it should be easy to extend
+        <code>permissions.query()</code> to accept a sequence&lt;&gt; too.
```
Comment (marcoscaceres) [inline index.html]: I don't think we should set this expectation. I think Promise.all is a great solution to this.

Comment (marcoscaceres) [inline index.html]: In fact, y u no add example of how awesome it is in action?

**Comment by marcoscaceres** [comment]
LGTM.

File: index.html (hunk 3 above)
Comment (mounirlamouri) [inline index.html]: I am going to open an issue for that.

**Pull request merged**